src/app/downloads/downloads-chart.component.ts

```diff
@@ -109,6 +107,8 @@ export class DownloadsChartComponent implements OnDestroy {
         // if we have episodes to combine with podcast total
         const episodeDatasets = this.episodeChartData.map(m => m.data);
         const allOtherEpisodesData: TimeseriesChartModel = {
+          // data: this.podcastChartData.data,
```

Comment:
can you remove this commented code?

Reply:
Whoops - will do.  I was experimenting with different colors/shades, and changing the graph type to "line", which required this other data.